src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+     * @param region    region of dynamo table
+     * @param tableName name of dynamo table
+     * @return s3 data node for output from dynamo table dump
+     */
+    static PipelineObject getS3BackupLocation(String s3Bucket, Region region, String tableName) {
+        String name = "S3BackupLocation-" + tableName;
+        String id = "S3BackupLocation-" + tableName;
+        String filename = "s3://" + s3Bucket + "/" + region.getName() + "/" + tableName +
+                "/#{format(@scheduledStartTime, 'YYYY-MM-dd-HH-mm-ss')}";
+
+
+        Field type = new Field().withKey("type").withStringValue("S3DataNode");
+        Field directoryPath = new Field().withKey("directoryPath").withStringValue(filename);
+        List<Field> fieldsList = Lists.newArrayList(type, directoryPath);
+
+        return new PipelineObject().withName(name).withId(id).withFields(fieldsList);
+    }
+
+    static List<PipelineObject> getEMRActivity(
+            TableDescription table, PipelineObject assignedCluster, Region region, String bucket) {
+        String tableName = table.getTableName();
+        String name = "TableBackupActivity-" + tableName;
+        String id = "TableBackupActivity-" + tableName;
+
+        PipelineObject inputTable = getDDBSourceTable(tableName, READ_THROUGHPUT_PERCENTAGE);
+
+        PipelineObject outputLocation = getS3BackupLocation(bucket, region, tableName);
+
+        String stepString = "s3://dynamodb-emr-" + region.getName() + "/emr-ddb-storage-" +
+                "handler/2.1.0/emr-ddb-2.1.0.jar,org.apache.hadoop.dynamodb.tools.DynamoDbExport," +
+                "#{output.directoryPath},#{input.tableName},#{input.readThroughputPercent}";
+
+        List<Field> fieldsList = Lists.newArrayList(
+                new Field().withKey("type").withStringValue("EmrActivity"),
+                new Field().withKey("input").withRefValue(inputTable.getId()),
+                new Field().withKey("output").withRefValue(outputLocation.getId()),
+                new Field().withKey("runsOn").withRefValue(assignedCluster.getId()),
+                new Field().withKey("maximumRetries").withStringValue("2"),
+                new Field().withKey("step").withStringValue(stepString)
+        );
+
+        PipelineObject activity = new PipelineObject().withName(name).withId(id).withFields(fieldsList);
+
+        return Lists.newArrayList(inputTable, outputLocation, activity);
+    }
+
+    static PipelineObject getEMRCluster(Region region) {
+        String name = "EmrClusterForBackup";
+        String id = "EmrClusterForBackup";
+
+        return new PipelineObject().withName(name).withId(id).withFields(
+                new Field().withKey("type").withStringValue("EmrCluster"),
+                new Field().withKey("amiVersion").withStringValue("3.8.0"),
+                new Field().withKey("masterInstanceType").withStringValue("m3.xlarge"),
+                new Field().withKey("coreInstanceType").withStringValue("m3.xlarge"),
```

Comment:
Our read/write capacities are small so m1.medium should work according to this
https://forums.aws.amazon.com/thread.jspa?messageID=667827

That said, I think m3.xlarge is the cheapest instance type recommend for production. But since this is a backup pipeline I think we should try m1.medium and change it as necessary.